dev-haskell/hoogle/hoogle-5.0.18.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+COMMIT="4bd6688857874fd41febb998a6420daa4b0b695f"
```

Comment:
If you are using snapshots you need to reflect it in package's version, like `hoogle-5.0.18_p20201010` or similar. Otherwise it will clash with actual release.

Reply:
The snapshot here is the same snapshot as the release tag.

The reason for using the snapshot rather than the published tarball is that the tarball does not contain everything needed to run tests. I have submitted a PR upstream for hlint (same author) to include such items in release tarballs, and depending on how that resolves  I can submit for hoogle as well.

In the meantime, being as this is the exact commit as the release, is it still proper to rename the version?

Reply:
Ah, no. The exact version is fine. Maybe adding a comment into ebuild would be enough.